Assets/Scripts/Quest/QuestMonster.cs

```diff
@@ -46,8 +49,8 @@ public QuestMonster(QuestData.UniqueMonster qm) : base()
         }
 
         // Read info from quest data or base type
-        info = EventManager.SymbolReplace(qm.info);
```

Comment:
But here it is a monster type defined by the quest, so it does need symbol replace.

I would like the quest translations at least (if not valkyrie translations) to use {symbol} notation on disk because it is readable in a text editor.